engine/src/events/callback/event_handler_interface.cpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by grzegorz on 08.05.19.
+//
+#include "event_handler_interface.h"
+
+namespace Bald {
+    std::vector<bool> EventHandlerInterface::m_TakenID;
+
+    EventHandlerInterface::EventHandlerInterface()
+            :
+            m_ID(0) {
```

Comment:
We have to follow the convention. I believe right now the convention is to do it like this:
Ctor()
: *tab* m_Var1(...)
  *tab* m_Var2(...) {
}